### PR TITLE
[zh-tw]: translate 5 HTML element

### PR DESCRIPTION
**Pull request by Dr-XYZ**

### CODE REVIEW

**Comment by Dr-XYZ** [comment]
Why need to edit the same? and where is wrong?

**Comment by yin1999** [comment]
> Why need to edit the same? and where is wrong?

Please preserve the EOF of each file, that means adding an empty line for each file.

File: files/zh-tw/web/html/element/article/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: "<article>: 文章內容元素"
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:2]: Please use a full-width colon and remove the quotes

```suggestion
title: <article>：文章內容元素
```

File: files/zh-tw/web/html/element/article/index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      <th scope="row">
+        <a href="/zh-TW/docs/Web/HTML/Content_categories"
+          >內容類型</a
+        >
+      </th>
+      <td>
+        <a href="/zh-TW/docs/Web/HTML/Content_categories#flow_content"
+          >流動內容</a
+        >、
+        <a
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:26]: The line break will produce a "space" between the punctuation and Chinese words ([translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E4%B8%AD%E6%96%87%E5%8F%8A%E6%8B%89%E4%B8%81%E8%AA%9E%E7%B3%BB%E6%96%87%E5%AD%97%E9%96%93%E4%B8%8D%E5%8A%A0%E4%BF%AE%E9%A3%BE%E7%A9%BA%E7%99%BD)):

```suggestion
        >、<a
```

File: files/zh-tw/web/html/element/article/index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    </tr>
+    <tr>
+      <th scope="row">標籤省略</th>
+      <td>{{no_tag_omission}}</td>
+    </tr>
+    <tr>
+      <th scope="row">允許的父元素</th>
+      <td>
+        任何接受
+        <a href="/zh-TW/docs/Web/HTML/Content_categories#flow_content"
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:51]: Translate the hash, and do not add line break here (see the above)
```suggestion
        任何接受<a
          href="/zh-TW/docs/Web/HTML/Content_categories#流內容"
```

File: files/zh-tw/web/html/element/article/index.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            >article</a
+          ></code
+        >
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">允許的 ARIA 角色</th>
+      <td>
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/application_role"><code>application</code></a>、<a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/document_role"><code>document</code></a>、
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/feed_role"><code>feed</code></a>、<a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/main_role"><code>main</code></a>、
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:71]: do not add line break

File: files/zh-tw/web/html/element/article/index.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+      <time datetime="2015-05-15 19:00">May 15</time>
+      by Staff.
+    </p>
+  </footer>
+</article>
+```
+
+## 結果
+
+{{EmbedLiveSample('Examples','','570')}}
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:140]: [translate the ID](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E7%BF%BB%E8%AD%AF%E6%A8%99%E9%A1%8C-id)

```suggestion
{{EmbedLiveSample('範例','','570')}}
```

File: files/zh-tw/web/html/element/article/index.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+## 規格
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:142]: see [common translation](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E5%B8%B8%E8%A6%8B%E6%A8%99%E9%A1%8C%E6%88%96%E8%A9%9E%E5%BD%99)

```suggestion
## 規範
```

File: files/zh-tw/web/html/element/article/index.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+{{Specifications}}
+
+## 瀏覽器相容性
+
+{{Compat}}
+
+## 參閱
```
Comment (yin1999) [inline files/zh-tw/web/html/element/article/index.md:150]:
```suggestion
## 參見
```

File: files/zh-tw/web/html/element/base/index.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
+---
+title: "<base>: 文件基礎 URL 元素"
+slug: Web/HTML/Element/base
+---
+
+{{HTMLSidebar}}
+
+**`<base>`** [HTML](/zh-TW/docs/Web/HTML) 元素指定用於文件中所有 _相對_ URL 的基礎 URL。一個文件中只能有一個 `<base>` 元素。
```
Comment (yin1999) [inline files/zh-tw/web/html/element/base/index.md:8]: Do not add spaces between Chinese words.

```suggestion
**`<base>`** [HTML](/zh-TW/docs/Web/HTML) 元素指定用於文件中所有*相對* URL 的基礎 URL。一個文件中只能有一個 `<base>` 元素。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
Hi @Dr-XYZ. I've listed some commom problem (for all those documents). Could you fix them before I have a deep check?

**Comment by Dr-XYZ** [comment]
Sure

**Review by yin1999 — APPROVED** [review]
Thank you @Dr-XYZ

**Pull request merged**